### PR TITLE
docs(installation): add CRA templates guide note

### PR DESCRIPTION
**Pull request by with-heart**
This PR adds a note to the Installation page pointing CRA users to the official CRA templates guide.

image

### CODE REVIEW

File: website/pages/docs/getting-started.mdx [hunk 1]
```diff
@@ -5,6 +5,9 @@ description: How to install and set up Chakra UI in your local project
 
 **First, you need to install Chakra UI**
 
+> If you're a `create-react-app` user, see the
```
Comment (ljosberinn) [inline website/pages/docs/getting-started.mdx:8]: > If you're using `create-react-app`, see the

Maybe just me, but `a ... user` feels odd

Comment (with-heart) [reply website/pages/docs/getting-started.mdx:8]: @ljosberinn How about "For `create-react-app` installation instructions," instead?

Comment (ljosberinn) [reply website/pages/docs/getting-started.mdx:8]: Fine by me, feels a bit formal, but I don't mind.

Comment (with-heart) [reply website/pages/docs/getting-started.mdx:8]: I agree but not sure how to state it better while keeping it short. I'll change it to that and if we come up with something better later, we can change it

**Pull request merged**